src/Altinn.AccessManagement.Integration/Clients/Altinn2RightsClient.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Altinn.AccessManagement.Core.Clients.Interfaces;
+using Altinn.AccessManagement.Core.Models;
+using Altinn.AccessManagement.Integration.Configuration;
+using Authorization.Platform.Authorization.Models;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.Rest.Azure;
+
+namespace Altinn.AccessManagement.Integration.Clients
+{
+    /// <summary>
+    /// Client for getting Altinn roles from AltinnII SBL Bridge
+    /// </summary>
+    public class Altinn2RightsClient : IAltinn2RightsClient
```

Comment:
Client implementation can be excluded from code coverage as they are mocked for our integration tests